bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaErrors.java

```diff
@@ -162,6 +163,10 @@ public static ErrorValue createNumericConversionError(Object inputValue, BType i
                 RuntimeErrors.INCOMPATIBLE_SIMPLE_TYPE_CONVERT_OPERATION, inputType, inputValue, targetType));
     }
 
+    public static ErrorValue createAsyncCallInsideLockError() {
```

Comment:
This method is only called in BLockStore.java right? Shall we move the method there?

Reply:
Fixed